pallets/phala/src/stakepool.rs

```diff
@@ -1219,6 +1257,11 @@ pub mod pallet {
 					user: user_info.user.clone(),
 					amount: reduced,
 				});
+				Self::deposit_event(Event::<T>::WithdrawalShare {
+					pid: pool_info.pid,
+					user: user_info.user.clone(),
+					share: withdrawing_shares,
```

Comment:
ditto. Also, please check if `withdrawing_shares` matches the exact value as it in `remove_stake`. Pay attention to the dust.